keps/sig-windows/windows-runtimeclass-support.md

```diff
@@ -333,6 +336,46 @@ message PullImageRequest {
 }
 ```
 
+#### Adding annotations to ImageSpec
+
+Current proposal is to change the `ImageSpec` to incorporate annotations for runtime class. These can be passed down to the container runtimes where various actions can be taken as per the runtime class specified.
+ 
+Currently ImageSpec contains just a string.
+
+```
+type ImageSpec struct {
+  Image string
+}
+```
+ 
+The proposal is to add Annotations into the ImageSpec:
+
+```
+type ImageSpec struct {
+  Image string
+  Annotations []Annotation
+}
+```
+ 
+The runtimeHandler annotation will be based on the Runtime Class specified by the user:
+```
+“runtimehandler”: “<corresponding values>”
```

Comment:
Can we also add `kubernetes.io` domain?

Reply:
as in `kubernetes.io/runtimehandler` ?

Reply:
those annotations are bit different scope though - they're stored in the container manifest. We're talking about changing CRI, not OCI

Reply:
right just a mention regarding another reserved annotation domain for these annotations.